src/XDGData/CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ if(NOT DEFINED APPDATA_RELEASE_DATE)
             WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
             OUTPUT_STRIP_TRAILING_WHITESPACE)
     else()
-        file(TIMESTAMP "${CMAKE_SOURCE_DIR}/CMakeLists.txt" APPDATA_RELEASE_DATE "%Y-%m-%d")
+        if(NOT (CMAKE_VERSION VERSION_LESS 3.8.0))
+            set(APPDATA_RELEASE_DATE ENV{SOURCE_DATE_EPOCH})
```

Comment:
`SOURCE_DATE_EPOCH` is an integer. I guess what you meant was:
```
if(NOT (CMAKE_VERSION VERSION_LESS 3.8.0))
    string(TIMESTAMP APPDATA_RELEASE_DATE "%Y-%m-%d")
else()
```

Reply:
Thanks for the tip! I checked using the freecad-daily package and the date in `/usr/share/freecad-daily/share/metainfo/org.freecadweb.FreeCAD.appdata.xml` was actually getting set to the string `"ENV{SOURCE_DATE_EPOCH}"`! I had just checked that the change made the packages reproducible and not what the actual value was getting set too, my bad.